README.md

```diff
@@ -0,0 +1,64 @@
+# Space-Travelers
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> It is a Single Page App (SPA) that display a list of Rockets, Mission, users can reserve rocket, and join into missions.
+
+
+## Built With
+![](https://img.shields.io/badge/-HTML-orange) ![](https://img.shields.io/badge/-CSS-blue) ![](https://img.shields.io/badge/-JavaScript-yellow) ![](https://img.shields.io/badge/-React-cyan) ![](https://img.shields.io/badge/-Redux-red)
+
+- HTML
+- CSS
+- Javascript
+- linters
+- webpack
+- React
+- Bootstrap
+- React-Bootstrap
+- Redux
+- Jest
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone repository in your local machine
+- run `npm install` in the terminal.
+- run `npn run start`.
+
+For testing you can follow these step:
+- run `npn run test`.
+
+## Live Demo
+
+[Coming soon!]()
+
+## Authors
+
+👤 **Ja'fer Yousef**
+
+- GitHub: [@githubhandle](https://github.com/jaferIdrees)
+- Twitter: [@twitterhandle](https://twitter.com/jafel_l)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/jaferll)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/jaferIdrees/space-travelers-hub/issues).
+
+## Show your support
+
+Give a ⭐️ to if you like this project!
+
+
+## Acknowledgments
+
+- Microverse
+- Code reviewer
+
+## 📝 License
+
+This project is [MIT](https://github.com/fahimy143/bookstroe-react-app/blob/Dev/LICENSE.md) licensed.
```

Comment:
- [OPTIONAL] Please ensure your readme file is visible on your repository, as this will enable visitors to get an idea of what your project is about and how to clone and run it.